### DISCUSSION TITLE
New StackBlitz template for v16 doesn't work

### DISCUSSION THREAD

**Discussion by psarno** [primefaces/community#320] [PrimeNG] [answered]
I am presented with the following when visiting the recently updated StackBlitz repro template link:

https://stackblitz.com/github/primefaces/primeng-issue-template

> Error in /turbo_modules/primeng@16.3.1/fesm2022/primeng-megamenu.mjs (154:159)
> Cannot access 'MegaMenu' before initialization

This was the same issue I ran into when trying to update the old v14 template to v16.

Now that we have an official update, why am I still seeing this error?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for reporting the issue, we've just updated version of PrimeNG. Fixed in here https://github.com/primefaces/primeng-issue-template/commit/578ef78acbda65a0c8122563cbcf086b6bbca6a2

**Reply by psarno** [reply]
@cetincakiroglu This version doesn't work for me either.

It shows an error in the console.

> Error: NG0908: In this configuration Angular requires Zone.js

I have no idea why this would be occuring.   As far as I can tell this has to do with the order in which the Angular JS bundles load in the index.html file, specifically the polyfills.

Are you not seeing the same?

![image](https://github.com/primefaces/community/assets/18640860/b13f33ab-530c-4d14-a9dd-463b147fa9b7)

**Comment by SoyDiego** [comment]
You can solve the problem importing `zone.js` @psarno @cetincakiroglu 

![image](https://github.com/primefaces/community/assets/19764334/46ca3a75-b707-4e55-a75b-9abc61864602)

**Comment by psarno** [comment]
@SoyDiego That does resolve it, thanks.

I am still left with the question of how the original issue of needing a new template was noted, the template was updated and a response was given, yet the template doesn't work as it is defined.

How did it get updated and considered done when it is broken?

We need a new link to a new official template with the proper zone.js reference now, because without that, every Github "New Issue" page is going to ask for a repro, and include a link that does not work.

It's either that or everyone is going to have to be aware of this particular necessary fix before they can submit a repro for an issue, which is probably not the best way to go.

**Reply by SoyDiego** [reply]
You are right @psarno.
I was looking If I was something in the project but I think the template come directly from Stackblitz.
So I think only Primetek Team can change it. But I agree with you about the template.

Thanks  for report and I hope theycan solve it soon.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for reporting the issue, and sorry for the misunderstanding. It's strange that the issue only occurs on stackblitz, that's why we've missed the import of zone.js. It's fixed right now here https://github.com/primefaces/primeng-issue-template/commit/2c0812932b813687aa1df270a243a703b627feb5
